website/notifications/model.py

```diff
@@ -0,0 +1,28 @@
+from modularodm import fields
+from framework.mongo import StoredObject, ObjectId
+from website.notifications.constants import NOTIFICATION_TYPES
+
+
+class Subscription(StoredObject):
+    _id = fields.StringField(primary=True)  # pxyz_wiki_updated, uabc_comment_replies
```

Comment:
It isn't readily clear to me what these comments are saying. Also, if we are using inline comments I believe PEP8 wants them to be two spaces after the end of the line.